packages/eslint-plugin/configs/recommended-with-formatting.js

```diff
@@ -14,6 +14,7 @@ const config = {
 		node: true,
 	},
 	globals: {
+		localStorage: true,
```

Comment:
Let's avoid exposing another global. In other places, it's done through `window.localStorage`.